validator/sawtooth_validator/journal/block_store.py

```diff
@@ -161,13 +151,11 @@ def _build_add_block_ops(self, blkw):
         """
         out = []
         blk_id = blkw.identifier
-        with self._commit_condition:
-            out.append((blk_id, blkw.block.SerializeToString()))
-            for batch in blkw.batches:
-                out.append((batch.header_signature, blk_id))
-                for txn in batch.transactions:
-                    out.append((txn.header_signature, blk_id))
-            self._commit_condition.notify_all()
+        out.append((blk_id, blkw.block.SerializeToString()))
```

Comment:
Other than the `_commit_condition`, this code is not mine. I believe you are correct that the tuples include blocks, batches, and txns, and the wait condition should never have been placed here. It's replacement (`_notify_observers`), is not going elsewhere.